packages/analytics-js-integrations/src/integrations/Clevertap/browser.js

```diff
@@ -58,7 +58,7 @@ class Clevertap {
     const sourceUrl =
       document.location.protocol === 'https:'
         ? 'https://d2r1yp2w7bby2u.cloudfront.net/js/clevertap.min.js'
-        : 'http://static.clevertap.com/js/clevertap.min.js';
+        : 'https://static.clevertap.com/js/clevertap.min.js';
```

Comment:
Does clevertap supports this?

Reply:
Please share the doc once

Reply:
@yashasvibajpai  can you explain this change ? Requirement was to always load https based SDK url and then verify on http site

Reply:
@utsabc [Here](https://developer.clevertap.com/docs/web-quickstart-guide#option-a-add-the-clevertap-javascript-library-manually-to-your-website) is the doc

Got it @krishna2020 fixing it

Reply:
fixed, seeing https send loading on the http test suite in the JS SDK